runner-integration-tests/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "runner-integration-tests"
+version = "0.1.0"
+authors = ["Reiner Dolp <reiner-dolp@users.noreply.github.com>"]
+edition = "2018"
+
+[dependencies]
+integration_test_codegen = { path = "../codegen-integration-tests" }
+difference = "2.0"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+lazy_static = "1.2.0"
+wait-timeout = "0.1.5"
```

Comment:
Do we need serde, lazy_static and wait-timeout in this crate?

Reply:
wait-timeout is used for `timeout` tests. `lazy_static` is used for caching cargo calls.
I think serde is indeed unused until #164 is merged.